ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueCommitting.scala

```diff
@@ -141,6 +143,30 @@ class KeyValueCommitting private[daml] (
         throw Err.InvalidSubmission("DamlSubmission payload not set")
     }
 
+  private def verifyStateUpdatesAgainstPreDeclaredOutputs(
```

Comment:
The purpose of this change is to be able to invoke `submissionOutputs` without having a `KeyValueCommitting` instance.